### PR TITLE
Feature/constracts

### PR DESCRIPTION
**Pull request by ReinaldoAguilar**

### CODE REVIEW

File: src/main/java/org/fundacionjala/sfdc/pages/contracts/ContractForm.java [hunk 1]
```diff
@@ -14,80 +16,68 @@
...
 
-    @FindBy(id = "CustomerSignedTitle")
-    @CacheLookup
-    private WebElement customerSignedTitle;
+    private WebElement accountNameLoopList;
 
-    @FindBy(id = "ctrc6")
-    @CacheLookup
-    private WebElement customerSignedDateTextField;
+    //add boton create
```
Comment (pablopzsj) [inline src/main/java/org/fundacionjala/sfdc/pages/contracts/ContractForm.java]: is button

File: src/main/java/org/fundacionjala/sfdc/pages/contracts/ContractForm.java [hunk 2]
```diff
@@ -148,8 +138,11 @@ public ContractForm setCustomerSignedBy(final String customerSignedBy) {
      * @return a contract form.
      */
     public ContractForm setAccountName(final String accountName) {
-        accountNameTextField.clear();
-        accountNameTextField.sendKeys(accountName);
+        accountNameTextField.click();
+        List<WebElement> myElements = driver.findElements(By.xpath("//div[contains(@class,'primaryLabel "
```
Comment (pablopzsj) [inline src/main/java/org/fundacionjala/sfdc/pages/contracts/ContractForm.java:137]: it wouldn't be better to use locator in the findElements method?

File: src/main/java/org/fundacionjala/sfdc/pages/contracts/ContractForm.java [hunk 3]
```diff
@@ -160,31 +153,12 @@ public ContractForm setAccountName(final String accountName) {
      * @return a contract form.
      */
     public ContractForm setCustomerSignedTitle(final String customerSignedTitle) {
-        customerSignedByTextField.clear();
-        customerSignedByTextField.sendKeys(customerSignedTitle);
-        return this;
-    }
+        accountNameTextField.click();
+        accountNameTextField.click();
+        List<WebElement> myElements = driver.findElements(By.xpath("//div[contains"
```
Comment (pablopzsj) [inline src/main/java/org/fundacionjala/sfdc/pages/contracts/ContractForm.java:153]: the same here

File: src/main/java/org/fundacionjala/sfdc/pages/contracts/ContractForm.java [hunk 4]
```diff
@@ -207,8 +181,10 @@ public ContractForm choosePriceBookType(final String type) {
      * @return a contract form.
      */
     public ContractForm chooseStatus(final String status) {
-        Select selectBox = new Select(statusMultiselect);
-        selectBox.selectByVisibleText(status);
+
+        statusMultiselect.click();
+        List<WebElement> myElements = driver.findElements(By.xpath("//a[contains(@role,'menuitemradio')]"));
```
Comment (pablopzsj) [inline src/main/java/org/fundacionjala/sfdc/pages/contracts/ContractForm.java:181]: the same here

File: src/main/java/org/fundacionjala/sfdc/pages/contracts/ContractForm.java (hunk 2 above)
Comment (ReinaldoAguilar) [reply src/main/java/org/fundacionjala/sfdc/pages/contracts/ContractForm.java:137]: pablo by way this method is necessary to display the list

Comment (ReinaldoAguilar) [reply src/main/java/org/fundacionjala/sfdc/pages/contracts/ContractForm.java:137]: I maybe Separate common methods

**Pull request merged**